.travis.yml

```diff
@@ -65,4 +65,5 @@ notifications:
     on_success: change
     on_failure: always
     rooms:
-      secure: MvPcu8ToJDcsDM5jIWugMcdRLjwbl7yDrZ6zDSmHamc7jETSt7dF0SbBIf5FoDGSeoHw/Yhxr/X38dh1eTqOYpCidOSpFKTAZwLQuMn02dj8yEC8YSfS3Of/sGTDpvY0BwqWKQnsllpAzOO1gNiqU+Bhvl6SmP3rl/RRoiggqUeLHdbRhTeYko1C3cYjFT3L45XgZxjGOFrFvEp+97NEE10+JEj5fFU7H6kzEvYMss61+soyHePGIDn8hxg26TAFtoCh1g+1o3x1D2XeKsh5TeDjLEUkXFvsgVg42NGl95P7xR1N61Dxw6op+gX2P2lVQKB+aM/sYxIoJx2swq+Kui5pu0Q9zFdCanfc5CQOekcXF+pAZPwWjq2xNaN/ne6pxuSwLRHXLCwi7g49fiDGwd0fudmsLX2xIXHTe5G91Kl4ESh6G2P/dCuLC/j0ZyBY+vhaQNC5bJthOxu/e6Wxb5n3Rqj+I0/j/DmX8aC3E7j8J6ByNrn6/VSlcwOVlpV2T811x/SioU00VJicw3mrILtsRoY0BXUNQgRUeiFBIjahlN9eGXxt5A4vzWzmGRNn29f18D9qMsVHNGbvolEFwXBis/V5AHW+YfuG29cWnyQWUyT7UqkyS4koWkdEYwr58zblHANAVlOQFaUmzuGaYyC0XDstenYOEN5PUHqgXg0=
+      # REVIEWDOG_GITHUB_API_TOKEN
+      secure: "Md83AEZnxZ/pXNEM4jrFO8vLVbWKW3SOzWqOCFFhsH/ojisQUF5SjMgH2ysZ/2J6AOps0w5+ny1ZAt0/Y2l64gX/5/qwbHERj/+f3kTKVXkGxOBYWCOdxdNlqYD90UJRG9HKqtcjNztd/JzoK+AZuE9rD9dKHOmmzLz/n6DsQEpu/jeDcn+rIrlRm7RrrKqNjbACnKEfMVL3WoSlOiKz6nrCq6Iyb9a3N1/UctaxSDR3nxZJ25E61C7tytFZkCx7uP1ep3Iok6uEdgpkPRmKUZUUOl1HR2O7dvi4HoNxWYut1xluhZ8Zbny/qOVJfBDnR5JRzgefzwKNTVUm97/NCKgaQcQXbNGieymErbi5bVnpFhcFxnqvrcpeT0drTw/ZTjD6bxpNnQLDXZeGUyEvLyEaNFreAI9LwADvzhpvi/xu1JoeLVGP4BfOlRj/v21PElS5kbK5FcPcEeTSqhJroOBe48p8dhlLI35U1/Y9KOBKS1Vlb117wpXlfFwMOhAKlXDdaUUGrQXwFb1nTAuo5WkPPgto0HBBK+mEtOUJLtEUZCfu4Zf1uH8PNzXSn6fhpSv36+Sklv+9obkqeJUCeEuoEJObhWrsVhGw/DcgzsRBmOtL+BcIpFDqYMKYrCAaOm+u2FIHGoCDE7ym2Xe+fw/K/XmSj/HsGcWNKkGZRso="
```

Comment:
@BupycHuk That's a Slack token. ReviewDog's is on line 17

Reply:
Yeah, i got it right after sending a commen. Then removed it :blush: